avBooth/accordion-option-directive/accordion-option-directive.less

```diff
@@ -27,7 +27,7 @@
   .answer-text,
   .answer-category,
   .answer-description {
-    word-break: break-all;
+    word-break: break-word;
```

Comment:
Can you please also add `hyphens: auto;` to everywhere where `word-break: break-word;` is being used?